cmake/FindGrackle.cmake

```diff
@@ -0,0 +1,60 @@
+# See LICENSE_ENZO file for license and copyright information
+
+#.rst:
+# FindGrackle
+# -------
+#
+# Finds the Grackle chemistry library
+#
+# This will define the following variables::
+#
+#   Grackle_FOUND    - True if the system has the Grackle library
+#
+# and the following imported targets::
+#
+#   Grackle::Grackle - The Grackle library
+
+find_path(Grackle_INCLUDE_DIR
+  NAMES grackle.h
+)
+
+# Grackle by default builds both dynamic and static libs.
+# We'll use the static one.
+find_library(Grackle_LIBRARY
+  NAMES grackle
+)
+
+# Ideally we'd also set a version, but I'm still looking for it in the file grackle installed
+#set(Grackle_VERSION 0.0.0)
```

Comment:
I just wanted to give you a heads up that I've given some thought on how to deal with this. I plan to deal with this in a future PR (unless you really want to do it)

A simple short-term solution is to write a test program (that we compile with CMake's `try_compile` command) that makes use of a new public API function (documented [here](https://grackle.readthedocs.io/en/latest/Integration.html#querying-library-version-information)). A more robust solution is to have Grackle install a pkg-config file or provide a [Config-file Package](https://cmake.org/cmake/help/latest/guide/using-dependencies/index.html#id3).

Reply:
I think the test program is an often used solution also for other packages and should be robust, too (as the actual code is being called).
I'll open an issue (ideally, `FindGrackle.cmake` should be provided by Grackle ;))